overviewmanger/src/main/java/nl/hu/bep3/groep2/overviewmanger/CustomMessage.java

```diff
@@ -0,0 +1,26 @@
+package nl.hu.bep3.groep2.overviewmanger;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public class CustomMessage {
```

Comment:
Ik zou niet gebruik maken van de CustomMessage, kijk even hoe dat gedaan wordt in MenuManger met events

Reply:
Done, alleen weet ik niet of de custom event key voor aanmaken van order en updaten van order goed is. Geloof het wel?